src/comps/ReclaimManager.tsx

```diff
@@ -0,0 +1,479 @@
+
+const LoadingInfo = () => {
+  return (
+    <FlowContainer>
+      <div className="flex h-full flex-col gap-2 items-center justify-center">
+        <div
+          className="inline-block h-8 w-8 animate-spin rounded-full border-4 border-solid border-current border-e-transparent align-[-0.125em] text-surface motion-reduce:animate-[spin_1.5s_linear_infinite] dark:text-orange"
+          role="status"
+        ></div>
+        <SubText>Loading Info</SubText>
+      </div>
+    </FlowContainer>
+  );
+};
+
+const NotFound = () => {
+  return (
+    <FlowContainer>
+      <>
+        <div className="w-full flex flex-row items-center justify-between">
+          <Heading>Transaction could not be found</Heading>
+        </div>
+        <div className="flex flex-1 ">
+          <div className="w-full p-4 bg-lightOrange h-24 rounded-lg flex flex-row items-center justify-center gap-2">
+            <InformationCircleIcon className="h-10 w-10 text-orange" />
+            <p className="text-orange font-Matter font-semibold text-sm break-keep">
+              The transaction you are looking for could not be found. Please
+              check the transaction details and try again. If you believe this
+              is an error please contact a team member.
+            </p>
+          </div>
+        </div>
+      </>
+    </FlowContainer>
+  );
+};
+
+const CantReclaim = () => {
+  return (
+    <FlowContainer>
+      <>
+        <div className="w-full flex flex-row items-center justify-between">
+          <Heading>Cannot Reclaim Deposit</Heading>
+        </div>
+        <div className="flex flex-1 ">
+          <div className="w-full p-4 bg-lightOrange h-24 rounded-lg flex flex-row items-center justify-center gap-2">
+            <InformationCircleIcon className="h-10 w-10 text-orange" />
+            <p className="text-orange font-Matter font-semibold text-sm break-keep">
+              This deposit cannot be reclaimed since it has been minted for sBTC
+              and deposited into the Stacks chain.
+            </p>
+          </div>
+        </div>
+      </>
+    </FlowContainer>
+  );
+};
+type ReclaimDepositProps = {
+  amount: number;
+  depositTransaction: EmilyDepositTransactionType;
+};
+
+const ReclaimDeposit = ({
+  amount,
+  depositTransaction,
+}: ReclaimDepositProps) => {
+  const { notify } = useNotifications();
+  const walletInfo = useAtomValue(walletInfoAtom);
+  const setShowWallet = useSetAtom(showConnectWalletAtom);
+  const router = useRouter();
+
+  const { WALLET_NETWORK: walletNetwork } = useAtomValue(bridgeConfigAtom);
+
+  const buildReclaimTransaction = async () => {
+    try {
+      // FIXME: move to env
+      const maxReclaimFee = 5000;
+
+      const btcAddress = getWalletAddress();
+
+      if (!btcAddress) {
+        return setShowWallet(true);
+      }
+
+      // FIXME: move to util or its own file
+      const unsignedTxHex = constructPsbtForReclaim({
+        depositAmount: amount * 1e8,
```

Comment:
We need to round this, otherwise when we try to convert it to bigint in`constructPsbtForReclaim` we may get an error like `The number 101200.00000000001 cannot be converted to a BigInt because it is not an integer`

Reply:
fixed